### PR TITLE
fix: high cpu usage

### PR DESCRIPTION
**Pull request by saledjenic**

### CODE REVIEW

**Review by pinebit — APPROVED** [review]

File: src/nim_status_client.nim [hunk 1]
```diff
@@ -137,6 +139,10 @@ proc mainProc() =
   engine.setRootContextProperty("browserModel", browserController.variant)
 
   proc changeLanguage(locale: string) =
+    if (locale == currentLanguageCode):
```
Comment (jrainville) [inline src/nim_status_client.nim:142]:
```suggestion
    if (locale === currentLanguageCode):
```

**Review by jrainville — APPROVED** [review]
Great job dude

File: src/nim_status_client.nim (hunk 1 above)
Comment (saledjenic) [reply src/nim_status_client.nim:142]: @jrainville is there a "===" in nim?

Comment (jrainville) [reply src/nim_status_client.nim:142]: No, I'm dumb. Ignore. 🙃

**Pull request merged**